python/lsst/faro/utils/separations.py

```diff
@@ -114,8 +116,8 @@ def magInRange(cat):
                 visit[obj2], ra[obj2], dec[obj2])
             if not distances:
                 if verbose:
-                    print("No matching visits found for objs: %d and %d" %
-                          (obj1, obj2))
+                    log.debug("No matching visits found for objs: %d and %d" %
```

Comment:
This and the one below need to do deferred string interpolation so that the logger can decide whether to bother. So change to:

```python
log.debug("Message %d and %d", obj1, obj2)
```